admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -205,7 +205,19 @@ export default class CreateOrderPage {
     );
 
     this.$container.on('change', createOrderMap.freeShippingSwitch, e =>
-      this.cartEditor.setFreeShipping(this.cartId, e.currentTarget.value),
+      this.cartEditor.updateDeliveryOptions(this.cartId),
+    );
+
+    this.$container.on('change', createOrderMap.recycledPackagingSwitch, e =>
+      this.cartEditor.updateDeliveryOptions(this.cartId),
+    );
+
+    this.$container.on('change', createOrderMap.isAGiftSwitch, e =>
+      this.cartEditor.updateDeliveryOptions(this.cartId),
+    );
+
+    this.$container.on('blur', createOrderMap.giftMessageField, e =>
```

Comment:
I'm listening for suggestions 😉 however keep in mind nobody noticed this was missing since 9 months in 177 Create an Order page so this extra level of being careful is maybe not needed for a feature so unused 😅

Reply:
Oh yeah I totally agree, and didn't expect you to improve this 🙂
I just noticed the potential issue, so if someone opens a bug we will be able to find it quickly 😛